orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/TargetServerGroupResolver.groovy

```diff
@@ -114,12 +114,14 @@ class TargetServerGroupResolver {
 
     def tsg = tsgs.find {
       (stage.context.regions && stage.context.regions.contains(it.getLocation().value)) ||
-        (stage.context.zones && stage.context.zones.contains(it.getLocation().value))
+        (stage.context.zones && stage.context.zones.contains(it.getLocation().value)) ||
```

Comment:
I think you can use:
def location = it.getLocation().value
return (stage.context.regions?.contains(location) || (stage.context.zones?.contains(location) || (stage.context.namespaces?.contains(location))